google/cloud/forseti/common/gcp_api/storage.py

```diff
@@ -686,3 +692,31 @@ def get_object_iam_policy(self, bucket, object_name, user_project=None):
                 'objectIamPolicy', e, 'bucket', bucket)
             LOGGER.error(api_exception)
             raise api_exception
+
+
+def upload_json(data, gcs_upload_path):
+    """Upload inventory summary/violation data in json format.
+
+    Args:
+        data (dict): the data to upload
+        gcs_upload_path (string): the GCS upload path.
+    """
+    with tempfile.NamedTemporaryFile() as tmp_violations:
```

Comment:
If this is to be a more general util method, then this should not be called violations

Reply:
Done.